README.md

```diff
@@ -314,6 +316,51 @@ Name            | Effect |
 `vagrant-box`    | Build the Vagrant box image using `packer` |
 `vagrant-prep`    | Shortcut for building everything needed for `make run` |
 
+# Deployment Customizations
+
+## Customize The Application Domain
+
+In a standard installation the domain used by applications pushed to
+CF is the same as the domain configured for CF itself.
+
+This document describes how to change this so that CF and applications
+use separate domains.
+
+:warning: The changes described below will work only for CAP 1.3.1,
+and higher.  They do not work for CAP 1.3, and will not work for the
+CAP 2 series to be.
+
+1. Follow the basic steps for deploying UAA and SCF.
+1. When deploying SCF add a section like
+
+   ```
+   bosh:
+     instance_groups:
+     - name: api-group
+       jobs:
+       - name: cloud_controller_ng
+         properties:
+           app_domains:
+           - APPDOMAIN
```

Comment:
Maybe `<APPDOMAIN>` so it is clearly a placeholder?

Reply:
ok

Reply:
Done with commit 05aaa3c57e9da760304a8f1cd5e48362695b77d0